packages/driver/cypress/integration/cypress/currentTest_spec.js

```diff
@@ -0,0 +1,53 @@
+let global_err = null
+
+try {
+  Cypress.currentTest
+} catch (e) {
+  global_err = e
+}
+
+describe('currentTest', () => {
+  let suite_err = null
+
+  try {
+    Cypress.currentTest
+  } catch (e) {
+    suite_err = e
+  }
+
+  before(() => {
+    expect(Cypress.currentTest.title).eq(cy.state('runnable').ctx.currentTest.title)
+  })
+
+  beforeEach(() => {
+    expect(Cypress.currentTest.title).eq(cy.state('runnable').ctx.currentTest.title)
+  })
+
+  after(() => {
+    expect(Cypress.currentTest.title).eq(cy.state('runnable').ctx.currentTest.title)
+  })
+
+  afterEach(() => {
+    expect(Cypress.currentTest.title).eq(cy.state('runnable').ctx.currentTest.title)
+  })
+
+  it('returns current test runnable', () => {
+    expect(Cypress.currentTest.title).eq(cy.state('runnable').title)
+  })
+
+  context('errors', () => {
+    it('throws if outside test', () => {
```

Comment:
any reason not to set `suite_err` in this `context`, closer to where it's really tested?